### PR TITLE
Check links with awesome_bot

### PR DESCRIPTION
**Pull request by OndraM**
Use https://github.com/dkhamsing/awesome_bot to check links; fix current links.

BTW @TomasVotruba if you merge this, don't forget to setup Travis builds for this repo and pull-requests :).

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Very nice tool! :clap:  Where did you find it? 

Thanks

File: README.md [hunk 1]
```diff
@@ -82,18 +82,17 @@ A curated list of amazingly awesome Symfony **articles, series and books**.
...
+* [Rich domain model with Symfony 2.5 and Doctrine 2.5](https://www.slideshare.net/_leopro_/rich-domain-model-with-symfony-25-and-doctrine-25) [2014]
 * [DDD with Symfony2: Folder Structure And Code First](http://williamdurand.fr/2013/08/07/ddd-with-symfony2-folder-structure-and-code-first/) [2013]
 
 
 ## General
 
 * [Lessons Learned from My First Symfony Project](http://www.thisprogrammingthing.com/2016/lessons-from-my-first-symfony-project/) [2016]
 * [Symfony best practices](http://blog.kevingomez.fr/2014/04/08/symfony-best-practices/) [2014]
-* [Symfony tips and tricks](http://www.slideshare.net/javier.eguiluz/symfony-tips-and-tricks) [2014]
-* [5 Ways to optimize Symfony Baseline Performance](https://tideways.io/profiler/blog/5-ways-to-optimize-symfony-baseline-performance) [2015]
```
Comment (TomasVotruba) [inline README.md:94]: Was this remove on purpose?

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
[Travis added](https://travis-ci.org/pehapkari/awesome-symfony-education/builds/225289443#L199)   :white_check_mark:

File: README.md (hunk 1 above)
Comment (OndraM) [reply README.md:94]: Yes, its a duplicate (duplicates are also reported by awesome bot by default) - or is it intentional to have this article here and also in the Performance section?

**Comment by OndraM** [comment]
Well I encountered it in awesome-php, when submitting  [Steward](https://github.com/lmc-eu/steward) - https://github.com/ziadoz/awesome-php/pull/753 :)

File: README.md (hunk 1 above)
Comment (TomasVotruba) [reply README.md:94]: All right then, thanks

**Pull request merged**